scripts/cli_artifacts.sh

```diff
@@ -16,9 +16,7 @@ do
   if [ -f "$CLI_BINARY_PATH" ]; then
       chmod +x $CLI_BINARY_PATH
       cp -v $CLI_BINARY_PATH ./artifacts_cli/swc-$BINDING_ABI
-      mv $CLI_BINARY_PATH ./scripts/npm/$BINDING_ABI
```

Comment:
I think this can break the CI pipeline

Reply:
How? This is moving swcx binary to npm pkg to be published, not the actual .node bindings.

Reply:
I can verify and report back.

Reply:
https://github.com/kwonoj/swc/actions/runs/3798606577/jobs/6460886033#step:9:110 CI passes publish (except real publish) with artifacts.

Reply:
Thank you!